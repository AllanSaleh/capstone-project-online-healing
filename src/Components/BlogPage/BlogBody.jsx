import React from 'react';

export default function BlogBody({ title, intro, subtitle, body }) {
  return (
    <div className="flex flex-col justify-evenly lg:h-section px-sides my-4">
      <div className="text-3xl lg:text-title">{title}</div>
      <div className="mt-4 lg:mt-0 text-lg lg:text-paragraph text-gray-700 text-justify">
        "{intro}"
      </div>
      <div className="mt-4 lg:mt-0 text-xl lg:text-subtitle">{subtitle}</div>
      <div className="mt-4 lg:mt-0 text-lg lg:text-paragraph text-gray-700 text-justify">
        {body.map((paragraph) => (
          <>
            <p>{paragraph}</p>
            <br />
          </>
        ))}
      </div>
    </div>
  );
}
