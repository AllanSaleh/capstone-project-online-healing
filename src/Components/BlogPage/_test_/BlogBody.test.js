import * as React from 'react';
import BlogBody from "../BlogBody";
import * as renderer from "react-test-renderer";

it('Blog Body renders correctly', () => {
  const tree = renderer
    .create(<BlogBody/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});