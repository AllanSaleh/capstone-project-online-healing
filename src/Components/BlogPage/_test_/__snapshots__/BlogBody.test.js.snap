// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-col justify-evenly lg:h-section px-sides my-4"
>
  <div
    className="text-3xl lg:text-title"
  >
    Mental Health and Technology
  </div>
  <div
    className="mt-4 lg:mt-0 text-lg lg:text-paragraph text-gray-700 text-justify"
  >
    Technology and mental well-being might seem like profoundly antagonistic terms. However, there are excellent digital tools to help you reach mindfulness and to practice meditation. You can even use smartphone apps, virtual reality solutions and digital devices as a form of relieving stressing and achieving digital detox – as, after a while, you will be able to practice all types of anti-anxiety skills on your own. So, let The Medical Futurist show you how you can enhance your mental and emotional health with the help of technology.
  </div>
  <div
    className="mt-4 lg:mt-0 text-xl lg:text-subtitle"
  >
    MENTAL HEALTH AND TECHNOLOGY
  </div>
  <div
    className="mt-4 lg:mt-0 text-lg lg:text-paragraph text-gray-700 text-justify"
  >
    “Just as we observe physical hygiene to stay well, we need to cultivate a kind of emotional hygiene too,” wrote the Dalai Lama on his Twitter-feed. There are three crucial points around His Holiness’ statement. First and foremost, the popularity of health-enhancing, physically active lifestyle is soaring worldwide. Nevertheless, there’s always room for improvement: the WHO says that globally, 23 percent of adults and 81 percent of school-going adolescents are not active enough. According to the organization, adults aged 18-64 should do at least 150 minutes of moderately intense physical activity each week, which is not unaccomplishable. Moreover, the numerically and statistically driven 21st-century mind could take a great inspiration and power from the various fitness trackers and wearables.
  </div>
  <div
    className="mt-4 lg:mt-0 text-lg lg:text-paragraph text-gray-700 text-justify"
  >
    Secondly, emotional hygiene seems to be thrown into the shade nowadays. Somehow, the culture of busyness prevents us from squeezing some time for mental health into our packed schedules. Information is pouring on us from millions of communication channels; we are connecting with hundreds of ‘friends’ on social media, we are obsessed with data due to the fear of missing out and time management slices up our days into meetings and to-do-lists.
  </div>
  <div
    className="mt-4 lg:mt-0 text-lg lg:text-paragraph text-gray-700 text-justify"
  >
    The pressure to perform and the stress to achieve is the air every busy-bee is breathing. However, turbulent lifestyle comes with a price. No wonder so many people have mental health issues. In the US, that means one in five adults – more than 40 million Americans! On the old continent, 27 percent of the population has to cope with mental disorders, which means 83 million struggling Europeans. In Asia, the prevalence of depression is 20 percent in Thailand, 19.9 percent in Taiwan, 19.4 percent in Korea, 17.5 in Malaysia and 16.5 percent in China.
  </div>
</div>
`;
